extension-ui/popup.html

```diff
@@ -2,27 +2,85 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <title>Website Blocker</title>
-  <link rel="stylesheet" href="popup.css">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Extension Popup</title>
+  <style>
+    body {
```

Comment:
1. move all styles to separate scss file
2. all colors should be defined in variables file
3. all values should be defined with % instead of px